src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

```diff
@@ -259,19 +259,20 @@ open class RsDefaultInsertHandler : InsertHandler<LookupElement> {
         context: InsertionContext,
         item: LookupElement
     ) {
-        val document = context.document
-        val startOffset = context.startOffset
-        val curUseItem = context.getElementOfType<RsUseItem>()
+        val document= context.document
+
         if (element is RsNameIdentifierOwner && !RsNamesValidator.isIdentifier(scopeName) && scopeName !in CAN_NOT_BE_ESCAPED) {
-            document.insertString(startOffset, RS_RAW_PREFIX)
+            document.insertString(context.startOffset, RS_RAW_PREFIX)
+            context.commitDocument() // Fixed PSI element escape
```

Comment:
I don't see problems with such approach
@vlad20012 could you please also review it